mungegithub/mungegithub.go

```diff
@@ -103,6 +106,33 @@ func main() {
 			if err := config.Features.Initialize(requestedFeatures); err != nil {
 				return err
 			}
+			if config.E2EStatusCheckOnly {
```

Comment:
I don't like this whole block. I'd much rather testing be done with `--pr-mungers=submit-queue --min-pr-number=0 --max-pr-number=0 --once`.  And get this output via a `--v=N` flag....

This is really uglying up a core function with very specific test only code...